addons/x_lib/functions/actors/fnc_createActor.sqf

```diff
@@ -25,34 +25,25 @@ See Also:
     - <ALIVE_fnc_sendActorMessage>
 
 Author:
-    Naught
+    Naught, dixon13
 ---------------------------------------------------------------------------- */
 
-private ["_actor", "_group"];
-_actor = [_this, 1, ["OBJECT"], objNull] call ALiVE_fnc_param;
-_group = [_this, 2, ["GROUP"], ALiVE_actors_mainGroup] call ALiVE_fnc_param;
+params [["_code", {}, [{}]], ["_actor", objNull, [objNull]], ["_group", grpNull, [grpNull]]];
 
-if (!isNil "_group") then
-{
-    if (isNull _actor) then
-    {
+if (_group == grpNull) then { _group = ALiVE_actors_mainGroup; };
```

Comment:
`grpNull` can't be compared with the `==` operator, either use `isNull` or `isEqualTo`.